### ISSUE TITLE
Is this a correct visualization?

### ISSUE THREAD

**Issue by gusevfe** [DessimozLab/iHam#3]
Hi!

I've made a simple orthoxml with just two species and two genes in a orthogroup. When visualising it using the code below, I get this: https://imgur.com/a/kAZy26u Is it correct? Why is there a vertical line between the two genes?

```
<!DOCTYPE html>
<html lang="en">
<head>
    <meta charset="UTF-8">
    <script src="https://d3js.org/d3.v3.js"></script>
    <script src="https://code.jquery.com/jquery-3.3.1.slim.min.js"></script>
    <link rel="stylesheet" href="iHam.css" type="text/css"/>
    <link href="https://fonts.googleapis.com/css?family=Roboto" rel="stylesheet">
    <link rel="stylesheet" href="http://tntvis.github.io/tnt/build/tnt.css" type="text/css" />
    <script src="http://tntvis.github.io/tnt/build/tnt.js" charset="utf-8"></script>
    <script src="https://dessimozlab.github.io/iHam/iHam.js"></script>
    <link rel="stylesheet" href="https://dessimozlab.github.io/iHam/iHam.css" type="text/css"/>
</head>
<body>

<div style="width: 1500px; min-width: 500px;" id="iham"></div>

<script>
  (function (div) {

    const data = {
      "tree": '(Mus musculus,Homo sapiens)Root',
      "orthoxml": '<orthoXML version="0.3">' +
        '<species name="Homo sapiens" NCBITaxId="9606">' + 
          '<database name="someDB" version="42">' + 
            '<genes>' + 
              '<gene id="1" geneId="hsa1" protId="hsa1" />' +
            '</genes>' + 
          '</database>' + 
        '</species>' + 
        '<species name="Mus musculus" NCBITaxId="10090">' + 
          '<database name="someDB" version="42">' + 
            '<genes>' + 
              '<gene id="2" geneId="mmu1"/>' + 
            '</genes>' + 
          '</database>' + 
        '</species>' + 
        '<groups>' + 
          '<orthologGroup>' + 
            '<geneRef id="1" />' + 
            '<geneRef id="2" />' + 
          '</orthologGroup>' + 
        '</groups>' + 
      '</orthoXML>',
      "fam_data": []
    }

    var theme = iHam()
      .orthoxml(data.orthoxml)
      .show_oma_link(false)
      .newick(data.tree)
      .fam_data(data.fam_data)
      .tree_width(330)
      .board_width(530);

    theme(div);
  })(document.getElementById('iham'));
</script>
</body>
</html>
```

**Comment by alpae** [comment]
No, it's not. Seems to be a bug in parsing the orthoxml properly.

**Comment by alpae** [comment]
This has been fixed with the upgrade to iham-parser 0.2.0
